backend/homework/7/src/main/java/org/handson/components/Tyre.java

```diff
@@ -0,0 +1,40 @@
+package org.handson.components;
+
+import org.springframework.beans.factory.annotation.Autowired;
+
+public enum Tyre {
```

Comment:
Good use of enum.

All constants/ENUMs and util functions could be shifted to a separate package respectively.